vms/platformvm/state/state.go

```diff
@@ -141,9 +139,11 @@ type State interface {
 
 	GetBlockIDAtHeight(height uint64) (ids.ID, error)
 
-	// ValidatorSet adds all the validators and delegators of [subnetID] into
-	// [vdrs].
-	ValidatorSet(subnetID ids.ID, vdrs validators.Set) error
+	// ApplyCurrentValidators adds all the current
+	// validators and delegators of [subnetID] into [vdrs].
+	// Should not be called repeatedly on the same [subnetID], as it will
+	// repeatedly add the same validators and delegators.
```

Comment:
q: so it's not idempotent? Maybe we should make it idempotent and drop this comment

Reply:
This doesn't make sense to be idempotent imo.